openkfw/TruBudget
# storage-service:Possibility to use azure blob storage

### Checklist




### Description
Implementation of Azure blob storage with local Azurite instance.

### How to test

1. Change env variable in storage service `STORAGE_PROVIDER=azure-blob`
2. Start local environment with `--use-azure-storage` e.g. `bash scripts/development/start-dev.sh --full --use-azure-storage --build`
3. Check if file functionality is working in app
4. ...




Closes #1656
